spec/controllers/cavc_remands_controller_spec.rb

```diff
@@ -55,8 +55,28 @@
 
     subject { post :create, params: params }
 
+    shared_examples "creates a remand depending on the sub-type" do
+      it "creates the CAVC remand and new appeal" do
+        remand_count = CavcRemand.count
+        cavc_count = Appeal.court_remand.count
+        subject
+
+        expect(response.status).to eq(201)
+        response_body = JSON.parse(response.body)
+
+        expect(response_body["cavc_remand"]["source_appeal_id"]).to eq(source_appeal.id)
+        expect(response_body["cavc_remand"]["decision_issue_ids"]).to match_array(decision_issue_ids)
+        expect(CavcRemand.count).to eq(remand_count + 1)
+
+        expect(response_body["cavc_appeal"]["appeal_id"]).not_to eq(source_appeal.id)
```

Comment:
Can we check what `response_body["cavc_appeal"]["appeal_id"]` should be equal to, rather than not equal to?

Reply:
So, I think what we're doing here is enough. We're checking "is the appeal ID different, the stream docket number the same, and the type remand" which feels sufficient to me to prove that we've created a new cavc appeal in the context of a Controller spec.

Reply:
Should `response_body["cavc_appeal"]["appeal_id"]` be `nil`?

Reply:
How about this:
```suggestion
        expect(response_body["cavc_appeal"]["id"]).to eq(CavcRemand.find(response_body["cavc_remand"]["id"]).remand_appeal_id)
```
I think you'll have to update this branch to get `remand_appeal_id` to be non-nil.